aa-sdk/core/src/middleware/erc7677middleware.ts

```diff
@@ -71,6 +95,8 @@ export type Erc7677MiddlewareParams<
     | TContext;
 };
 
+const Eth2Wei: bigint = 1_000_000_000_000_000_000n;
```

Comment:
this may be of use to you: https://viem.sh/docs/utilities/parseEther.html

Reply:
Also if you do need a constant, TypeScript convention is upper snake-case, e.g. `ETH_2_WEI`, although I would probably name this something like `WEI_PER_ETH` which is a big more specific.